client/landing/gutenboarding/available-designs.ts

```diff
@@ -49,9 +50,24 @@ export const getDesignImageUrl = ( design: Design ) => {
 		{
 			font_headings: design.fonts.headings,
 			font_base: design.fonts.base,
+			site_title: design.title,
```

Comment:
If we exclude `site_title`, we'd more often have previews generated already instead of needing to generate for each customer separately.

That said, if it works, would be _really cool_ to include it in thumbnails. 😁

Reply:
This is actually the theme title (maybe it wasn't previously?) I think I'll try using users title tomorrow, see how it goes, prefetching seems to work decently well at the moment.)

Reply:
Ahh, gotcha! Users title would be awesome of course — it'll just potentially generate a whole lot more mshots images once this is live to more people, so might be good to give systems heads up?